src/bin/cratesfyi.rs

```diff
@@ -554,28 +546,40 @@ enum DeleteSubcommand {
     },
 }
 
-struct Context {
+struct BinContext {
     build_queue: OnceCell<Arc<BuildQueue>>,
     storage: OnceCell<Arc<Storage>>,
     config: OnceCell<Arc<Config>>,
     pool: OnceCell<Pool>,
+    metrics: OnceCell<Arc<Metrics>>,
```

Comment:
`Metrics` has no cross-dependencies so could be a `Lazy` instead of a `OnceCell`

Reply:
I'd prefer to keep `OnceCell` for consistency sake with the rest of the others. Using `OnceCell` is also not as painful here as it is everywhere else, since we only access the underlying `OnceCell` in only one place.